content/repositories/creating-and-managing-repositories/creating-a-template-repository.md

```diff
@@ -25,7 +25,14 @@ To create a template repository, you must create a repository, then make the rep
 
 After you make your repository a template, anyone with access to the repository can generate a new repository with the same directory structure and files as your default branch. They can also choose to include all the other branches in your repository. Branches created from a template have unrelated histories, so you cannot create pull requests or merge between the branches. For more information, see "[AUTOTITLE](/repositories/creating-and-managing-repositories/creating-a-repository-from-a-template)."
 
+{% note %}
+
+**Note**: You can use a template repository on GitHub as starter code for an assignment on {data variables.product.prod_classroom}. For more information, see "[AUTOTITLE](/education/manage-coursework-with-github-classroom/teach-with-github-classroom/create-an-assignment-from-a-template-repository)."
```

Comment:
```suggestion
**Note:** You can use a template repository as starter code for an assignment on {% data variables.product.prod_classroom %}. For more information, see "[AUTOTITLE](/education/manage-coursework-with-github-classroom/teach-with-github-classroom/create-an-assignment-from-a-template-repository)."
```

This is a great use of a note, and the content looks great! 🎉 The only suggestions I'll make here are to remove "on GitHub" for concision, and to fix the variable—our variables always start `{% ` and end with ` %}` 🙂